vumi/persist/fake_redis.py

```diff
@@ -220,6 +220,16 @@ def delete(self, key):
             self._known_key_existence[key] = False
         return existed
 
+    @maybe_async
+    def rename(self, key, newkey):
+        if key == newkey:
+            raise ResponseError("source and destination objects are the same")
```

Comment:
Ah, so the test console in [redis.io](redis.io)'s docs is okay with this, but running this locally (v3.0.5), I get the error.